frontend/packages/knative-plugin/src/utils/__tests__/fetch-dynamic-eventsources-utils.spec.ts

```diff
@@ -42,9 +42,17 @@ describe('fetch-dynamic-eventsources: EventSources', () => {
   });
 
   it('should fetch models for duck type in case of error', async () => {
-    jest.spyOn(coFetch, 'coFetch').mockImplementation(() => Promise.reject(new Error('Error')));
+    // Suppress the warning to clean up the test output
+    jest.spyOn(console, 'warn').mockImplementation(jest.fn());
+
+    jest
+      .spyOn(coFetch, 'coFetch')
+      .mockImplementation(() => Promise.reject(new Error('Test Error')));
     await fetchEventSourcesCrd();
     expect(getEventSourceModels()).toHaveLength(0);
+
+    // eslint-disable-next-line no-console
+    (console.warn as any).mockRestore();
```

Comment:
I can remove this change if you want, but when testing the knative tests had a big error stack in the middle of it and I personally don't like seeing those in tests that are successfully passing.